msteams-platform/bots/calls-and-meetings/call-notifications.md

```diff
@@ -68,10 +72,17 @@ The OAuth token will have values like the following, and will be signed by Skype
 }
 ```
 
+The OpenID configuration published at <https://api.aps.skype.com/v1/.well-known/OpenIdConfiguration> can be used to verify the token. Each of the OAuth token values are used as follows:
```

Comment:
Each of the OAuth token values is used as follows
Each of the OAuth token value is used as follows
Each singular